torch/fx/node.py

```diff
@@ -197,6 +198,7 @@ def __init__(self, graph: 'Graph', name: str, op: str, target: 'Target',
                 annotation of values in the generated code or for other types
                 of analyses.
         """
+        super().__init__()
```

Comment:
Not passing any args to super?

Reply:
Init just sets erased, prev and next. I dont think I need to pass anything else.